coderd/database/databasefake/databasefake.go

```diff
@@ -2310,6 +2311,8 @@ func (q *fakeQuerier) InsertWorkspaceAgent(_ context.Context, arg database.Inser
 		StartupScript:        arg.StartupScript,
 		InstanceMetadata:     arg.InstanceMetadata,
 		ResourceMetadata:     arg.ResourceMetadata,
+		ConnectionTimeout:    arg.ConnectionTimeout,
+		TroubleshootingUrl:   arg.TroubleshootingUrl,
```

Comment:
nit: TroubleshootingURL

Reply:
I've tried to knock some out... you can rename them with `sqlc.yaml`.

Reply:
BTW I'm good with leaving this for further improvement.

Reply:
Fixed for the database 👍🏻, but not fixable for protobufs, sadly. 😔